### ISSUE TITLE
Data available to researchers

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#48656]
## User Story
As a VFS team researcher, I want to know what data is already available to teams supporting VA.gov so that I am able to access that data easily and do not waste my time trying to capture it on my own.

Assignee:
Peer Reviewer:

## Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

## Tasks
- [ ] _Placeholder task_

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] _Placeholder AC_

### Refinement
- [ ] Ticket has user story, description, tasks, and acceptance criteria
- [ ] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [ ] Ticket has been sized 
- [ ] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [ ] Ticket has been assigned to appropriate initiative or quarterly epic
- [ ] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by naomimarcussen** [comment]
Here's an initial list of resources for data that we can provide instructions on how to access.

- Vetdata (see [convo in research-ops channel](https://dsva.slack.com/archives/C0216PL32HJ/p1666375366412989))
- Medallia feedback 
- Medallia intercepts
- Domo salesforce issues (Ian)
- Domo Medallia report
- Vsignals

**Comment by shiragoodman** [comment]
@naomimarcussen the more I think about this, the more it makes sense that @michelle-dooley and the A&I team be involved. What do you think?

**Comment by naomimarcussen** [comment]
@shiragoodman Yes, I think that would be helpful, as I would need help in documenting how to access this data.

**Comment by shiragoodman** [comment]
@michelle-dooley could you look this over and see if it makes sense for your team to help?

**Comment by michelle-dooley** [comment]
@shiragoodman and @naomimarcussen - yes I should be involved.  So are you just looking for a list of specific reports a researcher may be interested in?

**Comment by naomimarcussen** [comment]
@michelle-dooley Yeah, a list of reports and how researchers can access those reports would be helpful. We can meet to discuss the list I started above to see if those make sense and note any others.

**Comment by shiragoodman** [comment]
I am removing the `Governance-team` label. If youd like us involved in this work, please let me know.

**Comment by michelle-dooley** [comment]
Hey @naomimarcussen - here is my list of [reports](https://docs.google.com/spreadsheets/d/1oD249Jpdn8_bPV8El7_2oDHHiaSeFJcXbq1bq0iJZsA/edit#gid=0) for UX Researchers.  Let me know if you have any questions.

**Comment by naomimarcussen** [comment]
Thank you, @michelle-dooley. This is great! I've added this list to a page that I'll publish to the Research and Design section of the Platform website. Take a look through it and make sure you're good with it. Where would you want to direct people if they need help with anything here?  https://vfs.atlassian.net/l/cp/FW12ga2a

**Comment by michelle-dooley** [comment]
Looks good.  Refer them to the #platform-analytics channel.
